authz/opa-bridge/policies/trino/main.rego

```diff
@@ -0,0 +1,53 @@
+package trino
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.trino
+
+default allow = false
+
+# ToDo: SetSchemaAuthorization, Test  "information_schema"
```

Comment:
should this be done before merging?

Reply:
Yes, it shows that the correct rule endpoint has been hit to the client.